courses/tests.py

```diff
@@ -9,22 +9,22 @@
 from .views import CourseView, AllCoursesView
 
 #Serializer Testing
-class AppUserSerializerTest(TestCase):
+class CourseSerializerTest(TestCase):
     @classmethod
     def setUp(self):
-        #build AppUser and AppUserSerializer instances
+        #build Course and CourseSerializer instances
         self.course_attributes = {
             "course_code": "SENG499",
-            "section": "A01",
+            "num_sections": 2,
             "course_title": "Design Project 2",
             "fall_offering": True,
             "spring_offering": True,
             "summer_offering": False,
-            "PENG_required": True,
+            "pengRequired": True,
```

Comment:
Smallest thing. Might want to update the pengRequired fields in the tests to match the expected JSON field. Like the 
comment you left on the courses serializer, ex {"fall": true, "spring": false, "summer": true}

Reply:
Good catch, thanks for finding this! Fixed.